src/settings-ui/Settings.UI.Library/ViewModels/PowerLauncherPluginViewModel.cs

```diff
@@ -180,14 +185,30 @@ public bool ShowNotAccessibleWarning
             get => !Disabled && !IsGlobal && string.IsNullOrWhiteSpace(ActionKeyword);
         }
 
-        private static readonly List<string> NotAllowedKeywords = new List<string>()
+        private static readonly List<string> ConflictingSimpleKeywords = new List<string>()
         {
-            "~", @"\", @"\\",
+            "~", @"\", @"\\", "%", // Folder path
```

Comment:
@davidegiacometti 
Adding `/` to the list too?

Reply:
Yes, make sense!